src/pages/_document.tsx

```diff
@@ -45,7 +45,26 @@ class MyDocument extends Document<DocumentProps> {
     render(): ReactElement {
         return (
             <Html lang="en" className="govuk-template app-html-class flexbox no-flexboxtweener">
-                <Head nonce={this.props.nonce} />
+                <Head nonce={this.props.nonce}>
+                    {process.env.NODE_ENV === 'production' && (
```

Comment:
We don't want to check NODE_ENV since that will be production on test, pre-prod and prod, there's an env var named `STAGE` which we need to check is equal to `prod`